recipes/shared-memory-dict/meta.yaml

```
@@ -0,0 +1,78 @@
+{% set name = "shared-memory-dict" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}-meta
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/shared-memory-dict-{{ version }}.tar.gz
+  sha256: 8294257e1a2466afb5e19a878a8c446325b8913d57d6d9578ccba905564a974d
+
+build:
+  number: 0
+
+outputs:
+  - name: shared-memory-dict
+    build:
+      noarch: python
+    script: build_base.sh
+    requirements:
+      host:
+        - pip
+        - python >=3.8,<4.0
+        - poetry
+      run:
+        - python >=3.8,<4.0
```

```suggestion
      run:
        - python >=3.8,<4.0
      run_constrained:
        - aiocache >=0.11.1,<0.12.0
        - django >=3.0.8,<4.0.0
```
You actually need to specify these version constraints here and not in the metapackages because if these packages are installed manually, they still need to meet the constraints.